Documentation/ApiOverview/Localization/LocalizationApi/LanguageService.rst

```diff
@@ -6,18 +6,21 @@ LanguageService
 ===============
 
 This class is used to translate strings in plain PHP. For examples
-see :ref:`extension-localization-php`. A :php:`LanguageService` **should not**
+see :ref:`extension-localization-php`. A
+:php-short:`\TYPO3\CMS\Core\Localization\LanguageService` **should not**
 be created directly, therefore its constructor is internal. Create a
-:php:`LanguageService` with the :ref:`LanguageServiceFactory-api`.
+:php-short:`\TYPO3\CMS\Core\Localization\LanguageService` with the
+:ref:`LanguageServiceFactory-api`.
 
 In the backend context a :php:`LanguageService` is stored in the global
 variable :php:`$GLOBALS['LANG']`.
-In the frontend a :php:`LanguageService` can be accessed via the contentObject:
+In the frontend a :php-short:`\TYPO3\CMS\Core\Localization\LanguageService` can
```

Comment:
In CLI $GLOBALS['LANG'] is also not available and has to be instantiated individually